README.adoc

```diff
@@ -1,10 +1,7 @@
-= Stackable Open Policy Agent Operator
+= Stackable Operator for OpenPolicyAgent
 
-This Kubernetes Operator deploys and manages the Open Policy Agent (OPA).
+This is a Kubernetes Operator to manage OpenPolicyAgent servers.
```

Comment:
I think we should use the shortcut OPA somewhere on top. I think many people can relate better to the shortcut (at least in my case).

Reply:
I've changed it